build-system/pr-check/npm-checks.js

```diff
@@ -96,13 +96,12 @@ function isPackageLockFileProperlyUpdated() {
 
 /**
  * Runs both npm checks, and returns false if either one fails.
- * @param {string} filename
  * @return {boolean}
  */
-function runNpmChecks(filename) {
+function runNpmChecks() {
   return (
-    isPackageLockFileInSync(filename) &&
-    isPackageLockFileProperlyUpdated(filename)
+    isPackageLockFileInSync() &&
+    isPackageLockFileProperlyUpdated()
```

Comment:
It's unclear how this is a noop

Reply:
Neither of those functions accepts any arguments.

Reply:
Yep, these args should go away. The fact that they're here is my fault 😃